### PR TITLE
Added NavigationLink on SearchResult

### PR DESCRIPTION
**Pull request by minkwonSQ**
Weird, I am not sure why xcscheme got included here

### CODE REVIEW

File: KVMarketSquare.xcodeproj/xcshareddata/xcschemes/KVMarketSquare.xcscheme [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (KVTaniguchi) [inline KVMarketSquare.xcodeproj/xcshareddata/xcschemes/KVMarketSquare.xcscheme:1]: theres gotta be a way to tell github to ignore these, i know we can gitignore them locally but would be better here 🤷

File: KVMarketSquare/Scenes/SearchView.swift [hunk 2]
```diff
@@ -152,26 +152,20 @@ struct SellerResultsListView: View {
         switch task.result {
         case .success(let model):
             List(model.data) { store in
-                Button {
-                    let sellerStore = SellerAppData(
+                NavigationLink {
+                    StoreWebView(store: SellerAppData(
```
Comment (KVTaniguchi) [inline KVMarketSquare/Scenes/SearchView.swift:156]: i am going to guess this has something to do with the way this view immediately tries to load the task on init - we can move that into the .none of the body result case in this view

Comment (KVTaniguchi) [reply KVMarketSquare/Scenes/SearchView.swift:156]: i will do that in a pr in a minute, hold on

**Comment by KVTaniguchi** [comment]
ah shoot i commited the xcsharedata 🤦

File: KVMarketSquare/Scenes/SearchView.swift (hunk 2 above)
Comment (minkwonSQ) [reply KVMarketSquare/Scenes/SearchView.swift:156]: I think I tried to load it from .none once but did not dive deeper. let me know how this goes 👍

**Pull request closed without merging**